src/model/modelinfo/ElementFactory.ts

```diff
@@ -0,0 +1,136 @@
+import Element from "./Element";
+import ChoiceElement, { ChoiceType } from "./ChoiceElement";
+import ListElement from "./ListElement";
+import SimpleElement from "./SimpleElement";
+import { normalizeElementTypeName } from "../../utils";
+
+/**
+ * This is how the XML is structured for an ElementTypeSpecifier's choice option.
+ * Example:
+ * <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="FHIR" name="CodeableConcept"/>
+ *
+ * It could also be how the XML is structured for the ListTypeSpecifier's NamedTypeSpecifier child:
+ * <ns4:elementTypeSpecifier xsi:type="ns4:ListTypeSpecifier">
+ *   <ns4:elementTypeSpecifier xsi:type="ns4:NamedTypeSpecifier" modelName="FHIR" name="ClaimResponse.Item"/>
+ * </ns4:elementTypeSpecifier>
+ */
+export interface ISimpleSpecifierXml {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)